contents/docs/tutorials/b2b.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Tracking Key B2B Product Metrics
+sidebar: Docs
+showTitle: true
+---
+
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 10 minutes ☕☕☕_</small>
+
+<br />
+
+<span class="larger-image">
+
+![B2B Tutorial Banner Image](../../images/tutorials/banners/b2b.png)
+
+</span>
+
+<br />
+
+While there is some overlap between the analytics processes of B2C and B2B businesses, there are significant differences in the business model that need to be accounted for when tracking key performance metrics.
+
+As such, this tutorial will walk you through how to track usage of your B2B product, going over key metrics and how you can set up your analytics views to reflect them.
+
+### Pre-Requisites
+
+To follow this tutorial along, you need to:
+
+1. Have [deployed PostHog](/docs/deployment).
+1. Have started receiving events via our [snippet](/docs/integrations/js-integration), one of our [integrations](/docs/integrations), or our [API](/docs/api/overview)
+
+> **Pro Tip:** To make the most out of this tutorial, we recommend that you have some [actions](/docs/tutorials/actions#sorting-through-your-events-with-actions) set up, as well as have tried sending some [custom events](/docs/tutorials/actions#using-custom-events-to-track-advanced-behaviors). These are explained in detail in our [Complete Guide to Event Tracking](/docs/tutorials/actions). Additionally, it is also beneficial to be familiar with [Cohorts](/docs/tutorials/cohorts).
+ 
+
+### Determining Product-Market Fit
+
+The first question you should seek an answer to is: _Is my product being used?_
+
+Given that the answer is a _yes_, you should then ask: _Who's using it?_
```

Comment:
I'd skew these towards the article title more heavily. The kind of questions a B2B saas company will have are:

* We've set a customer up on a trial. Have they logged in yet? Have they added more users?
* We've set a customer up on a trial, are they using the main features? Which users have tried it and which haven't?
* I need to know if a specific customer is likely to churn. Are they using the product?
* How can I easily spot which customers are likely not to renew?
* Overall, how strong is our product market fit - are customers repeatedly using the product over time?